content/en/llm_observability/monitoring/metrics.md

```diff
@@ -40,6 +40,18 @@ After you instrument your application with LLM Observability, you can access LLM
 |-------------|-------------|-------------|------|
 | `ml_obs.span.embedding.input.tokens` | Number of input tokens used for generating an embedding | Distribution | `env`, `error`, `ml_app`, `model_name`, `model_provider`, `service`, `version` |
 
+### LLM cost metrics
+
+| Metric Name | Description | Metric Type | Tags |
+|-------------|-------------|-------------|------|
+| `ml_obs.span.llm.input.cost` | Estimated input cost in an LLM span | Distribution | `env`, `error`, `ml_app`, `model_name`, `model_provider`, `service`, `version` |
+| `ml_obs.span.embedding.input.cost` | Estimated input cost in an embedding span | Distribution | `env`, `error`, `ml_app`, `model_name`, `model_provider`, `service`, `version` |
```

Comment:
```suggestion
| `ml_obs.span.embedding.input.cost` | Estimated input cost in an embedding span | Distribution | `env`, `error`, `ml_app`, `model_name`, `model_provider`, `service`, `version`, `source` |
```
We're in the middle of adding a `source` tag for cost metrics which indicate either "user" or "auto" to differentiate between user-provided and computed cost metrics